checkstyle-tester/codenarc.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 @Grapes(
-@Grab('org.codenarc:CodeNarc:1.4')
+@Grab('org.codenarc:CodeNarc:1.5')
```

Comment:
Needed to do this since our groovy version changed (image version is 3.0.9, version we installed in travis was 2.4.17), related to this bug: https://github.com/CodeNarc/CodeNarc/issues/426

Reply:
Issue created to update to latest version: https://github.com/checkstyle/contribution/issues/757